apps/dashboard/src/components/side-navigation/side-navigation.tsx

```diff
@@ -92,54 +39,61 @@ export const SideNavigation = () => {
-          </NavigationGroup>
-          <NavigationGroup label="Developer">
-            <NavigationLink to={LEGACY_ROUTES.INTEGRATIONS} isExternal>
-              <RiStore3Line className="size-4" />
-              <span>Integration Store</span>
-            </NavigationLink>
-            <NavigationLink to={LEGACY_ROUTES.API_KEYS} isExternal>
-              <RiKey2Line className="size-4" />
-              <span>API Keys</span>
-            </NavigationLink>
-          </NavigationGroup>
-          <NavigationGroup label="Application">
-            <NavigationLink to={LEGACY_ROUTES.SETTINGS} isExternal>
-              <RiSettings4Line className="size-4" />
-              <span>Settings</span>
-            </NavigationLink>
-          </NavigationGroup>
-          <NavigationGroup>
-            <NavigationLink to={LEGACY_ROUTES.INVITE_TEAM_MEMBERS} isExternal>
-              <RiUserAddLine className="size-4" />
-              <span>Invite teammates</span>
-            </NavigationLink>
-          </NavigationGroup>
+        <nav className="flex h-full flex-1 flex-col">
+          <div className="flex flex-col gap-4">
+            <NavigationGroup>
+              <NavigationLink to={buildRoute(ROUTES.WORKFLOWS, { environmentSlug: currentEnvironment?.slug ?? '' })}>
+                <RiRouteFill className="size-4" />
+                <span>Workflows</span>
+              </NavigationLink>
+              <SubscribersStayTunedModal>
+                <span onClick={() => track(TelemetryEvent.SUBSCRIBERS_LINK_CLICKED)}>
+                  <NavigationLink>
+                    <RiGroup2Line className="size-4" />
+                    <span>Subscribers</span>
+                  </NavigationLink>
+                </span>
+              </SubscribersStayTunedModal>
+            </NavigationGroup>
+            <NavigationGroup label="Monitor">
+              <NavigationLink to={LEGACY_ROUTES.ACTIVITY_FEED} isExternal>
+                <RiBarChartBoxLine className="size-4" />
+                <span>Activity Feed</span>
+              </NavigationLink>
+            </NavigationGroup>
+            <NavigationGroup label="Developer">
+              <NavigationLink to={LEGACY_ROUTES.INTEGRATIONS} isExternal>
+                <RiStore3Line className="size-4" />
+                <span>Integration Store</span>
+              </NavigationLink>
+              <NavigationLink to={LEGACY_ROUTES.API_KEYS} isExternal>
+                <RiKey2Line className="size-4" />
+                <span>API Keys</span>
+              </NavigationLink>
+            </NavigationGroup>
+            <NavigationGroup label="Application">
+              <NavigationLink to={LEGACY_ROUTES.SETTINGS} isExternal>
+                <RiSettings4Line className="size-4" />
+                <span>Settings</span>
+              </NavigationLink>
+            </NavigationGroup>
+          </div>
+
+          <div className="mt-auto gap-8 pt-4">
+            <FreeTrialCard />
```

Comment:
Removed absolute positioning of the free trial card, to be position above invite members when availablle